tmt/base.py

```diff
@@ -536,22 +535,17 @@ def export(self, format_='yaml', keys=None):
                 data['environment'] = ' '.join(
                     tmt.utils.shell_variables(self.environment))
             return data
-
         # Export to Nitrate test case management system
         elif format_ == 'nitrate':
+            # FIXME: should return data related to 'nitrate'
             tmt.export.export_to_nitrate(self)
-
         # Common node export otherwise
-        elif keys == 'fmf-id':
-            if format_ == 'dict':
+        elif formate_ == 'dict' or formate_ == 'yaml':
```

Comment:
This seems to be a typo, `formate_` vs `format_`.

Reply:
thank you, @happz, the typo is fixed.